setup.cfg

```diff
@@ -1,4 +1,61 @@
 [metadata]
-description_file = README.md
```

Comment:
There is no mention of this option in the setuptools docs. This seems to be redundant/replaced by `long_description = file:...`. According to [this Stackoverflow comment](https://stackoverflow.com/questions/60084128/does-description-file-in-setup-cfg-section-metadata-have-any-effect#comment108118088_60084977), this field was specific to PBR

Reply:
This has been there since the beginning, I never questioned it... but you are right.

While at it, maybe we should also add an `pyproject.toml` file?

Reply:
> While at it, maybe we should also add an `pyproject.toml` file?

What do you think we should put in it? Should I just set the `build-system` key?

Reply:
Yes, I think this would be sufficient for the time being - thanks!